.ci/run_metrics_PR_ci.sh

```diff
@@ -34,10 +34,7 @@ source "${SCRIPT_DIR}/../metrics/lib/common.bash"
 
 KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
 
-# Density paramter to set the number of iterations of the test
-DENSITY_FORMAT_RESULTS="${DENSITY_FORMAT_RESULTS:-json}"
-
-# Density parameters to select between 'csv' and 'json' results format
+# Num of repetitions for the 'memory inside container' metrics check.
 DENSITY_TEST_REPETITIONS="${DENSITY_TEST_REPETITIONS:-1}"
```

Comment:
not exactly related to this change, but realized that you are expecting `.ci/run_metrics_PR.sh` to be called with one argument, but actually it is being called with no arguments, check: https://github.com/kata-containers/tests/blob/main/.ci/run.sh#L136

I think that as this is a variable needed for a specific test (and this scripts launches several tests), it would be better to not get the value from an argument of this script.